inc/invitation.class.php

```diff
@@ -203,10 +216,19 @@ public function prepareInputForUpdate($input) {
       return $input;
    }
 
+   /**
+    * Finds the invitation that matches the token given in argument
+    * @param string $token
+    * @return boolean true if the invitation token exist
+    */
    public function getFromDBByToken($token) {
       return $this->getFromDBByQuery("WHERE `invitation_token`='$token'");
    }
 
+   /**
+    * Generates the Invitation Token
+    * @return string the generated pseudo random bytes
```

Comment:
"pseudo random bytes" it too specific to the implementation. "Random token" or "token" is sufficient.